src/cascade/executor/epiviz_runner.py

```diff
@@ -290,6 +295,8 @@ def model_context_from_settings(execution_context, settings):
      7. Construct all Random Effects.
     """
     model_context = initial_context_from_epiviz(settings)
+    model_context.parameters.location_id = execution_context.parameters.location_id
```

Comment:
I would do it, but it looks like the execution context holds what the plan creates. This will be so much clearer for the the refactor. We have so many decisions to make.